cms/static/js/spec/views/modals/move_xblock_modal_spec.js

```diff
@@ -44,9 +43,209 @@ define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpe
                     renderViewsSpy;
                 showModal();
                 renderViewsSpy = spyOn(modal, 'renderViews');
-                AjaxHelpers.expectRequest(requests, 'GET', OUTLINE_URL);
+                AjaxHelpers.expectRequest(requests, 'GET', outlineUrl);
                 AjaxHelpers.respondWithJson(requests, {});
                 expect(renderViewsSpy).toHaveBeenCalled();
             });
         });
+
+        showModal = function() {
+            modal = new MoveXBlockModal({
+                sourceXBlockInfo: new XBlockInfo({
+                    id: sourceLocator,
+                    display_name: sourceDisplayName,
+                    category: 'html'
+                }),
+                sourceParentXBlockInfo: new XBlockInfo({
+                    id: sourceParentLocator,
+                    display_name: 'VERT 101',
+                    category: 'vertical'
+                }),
+                XBlockUrlRoot: '/xblock',
+                outlineURL: outlineUrl
+            });
+            modal.show();
+        };
+
+        selectTargetParent = function(parentLocator) {
+            modal.moveXBlockListView = {
+                parent_info: {
+                    parent: {
+                        id: parentLocator
+                    }
+                },
+                remove: function() {}   // attach a fake remove method
```

Comment:
This was added because we needed to test the move functionality on sandbox.

We added this method because we are mocking `moveXBlockListView`, I didn't want to make call to the actual `moveXBlockListView` which would require extra arguments and would more complex. 

But, ultimately this needs to be changed in Enable/Disable Move ( we would get `targetParentInfo` in a better way and in this test change this method accordingly ...